cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,98 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
+import {
+  updateSuccessMessage,
+  confirmationMessage
+} from '../plugins/alertMessages';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
```

Comment:
It's recommended to avoid direct database manipulation such as 'db:clear' in tests.

Reply:
You wrote:
The 'db:clear' task is run before each test suite but not before each test. This could lead to tests affecting each other and causing false positives/negative
[https://github.com/mate-academy/qa_cypress_e2e_project/pull/59#discussion_r1291103375](url)